src/components/home-page/past-events.jsx

```diff
@@ -28,35 +29,41 @@ const injectResponsiveOption = option => ({ responsiveOptions: config }) => {
 }
 
 let PastEventsContainer = styled.section`
-  padding: 6rem 2rem;
-  ${media.tablet`padding: 9rem 7rem;`};
-  ${media.desktop`padding: 10rem;`};
+  padding-bottom: 6rem;
+  ${media.tablet`padding-bottom: 7rem;`};
+  ${media.desktop`padding-bottom: 10rem`};
+`
+
+let HeaderPaddings = styled.div`
+  padding: 6rem 2rem 0 2rem;
```

Comment:
you can use `<Box p={[...]}>` from `grid-styled`

Reply:
Oh cool, I've never used it before, but I am definitely gonna check it and change it to box :smile: 

Also I've changed all const to let inside this file :+1:

Reply:
fixed